tap.go

```diff
@@ -1,24 +1,34 @@
 package tap
 
+import "io"
+import "os"
 import "fmt"
 import "testing/quick"
 
 type T struct {
 	nextTestNumber int
+	outputFile     *os.File
 }
 
 // New creates a new Tap value
-func New() *T {
+func New(fileName string) *T {
```

Comment:
This change would break tests for all existing users.  If we want output to go somewhere other than `os.Stdout` we'll need a different function.  Ideally it would be able to send output to any `io.Writer`.  So maybe we'd want a function like `Fnew(w io.Writer)` (with the leading `F` modeled on similar functions like [fmt.Fprintf](https://golang.org/pkg/fmt/#Fprintf)).  Although I'm open to other name suggestions.

Anyway, `New` would just become a wrapper around that function like this:

```go
func New() *T {
    return Fnew(os.Stdout)
}
```